### PR TITLE
style: improve cancellation string

### PR DESCRIPTION
**Pull request by milkshakeiii**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: bigframes/formatting_helpers.py [hunk 1]
```diff
@@ -155,7 +155,9 @@ def wait_for_query_job(
         raise
     except KeyboardInterrupt:
         query_job.cancel()
-        print(f"Requested cancelation for {query_job.job_type} {query_job.job_id}...")
+        print(
+            f"Requested cancellation for {query_job.job_type.capitalize()} job {query_job.job_id}..."
```
Comment (tswast) [inline bigframes/formatting_helpers.py:159]: Nit: we should also include the job location. It's common for folks to give support this job ID but without the location it's not easy to look up where this job's logs are.

Comment (milkshakeiii) [reply bigframes/formatting_helpers.py:159]: It looks like https://cloud.google.com/python/docs/reference/bigquery/latest/google.cloud.bigquery.job.QueryJob#google_cloud_bigquery_job_QueryJob_location is not very specific (for example: "US"). Do you think that's sufficient? Thanks.

Comment (tswast) [reply bigframes/formatting_helpers.py]: Yes, that's correct. "US" means our US-multi-region. For other regions in the US, it'll be a string specific to that region, such as "us-central1"

**Review by tswast — APPROVED** [review]

File: bigframes/formatting_helpers.py (hunk 1 above)
Comment (milkshakeiii) [reply bigframes/formatting_helpers.py]: Nice, I hoped that was the case but the documentation was very minimal

"
location
str: Location where the job runs.
"

Ha.

Thanks.

**Pull request merged**